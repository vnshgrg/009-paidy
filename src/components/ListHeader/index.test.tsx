import "react-native";
import { Text } from "react-native";
import React from "react";
import { ListHeader } from "./index";

import renderer from "react-test-renderer";

it("renders ListHeader correctly", () => {
  const tree = renderer
    .create(<ListHeader stats={{ total: 2, completed: 1 }} onLock={() => {}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
