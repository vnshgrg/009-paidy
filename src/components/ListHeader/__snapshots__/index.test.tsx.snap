// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ListHeader correctly 1`] = `
<View
  style={
    {
      "alignItems": "flex-start",
      "backgroundColor": "#f6f6f6",
      "borderBottomColor": "#eee",
      "borderBottomWidth": 1,
      "flexDirection": "row",
      "padding": 12,
    }
  }
>
  <View
    style={
      {
        "flexGrow": 1,
      }
    }
  >
    <Text
      style={
        {
          "color": "#333",
          "fontSize": 24,
          "fontWeight": "bold",
          "marginBottom": 4,
        }
      }
    >
      Todo
    </Text>
    <Text
      style={
        {
          "color": "#666",
          "fontSize": 12,
        }
      }
    >
      1
       of 
      2
       completed
    </Text>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "height": "100%",
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#eee",
          "borderRadius": 100,
          "flexDirection": "row",
          "marginHorizontal": 6,
          "opacity": 1,
          "paddingHorizontal": 10,
          "paddingVertical": 10,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="20"
        bbWidth="20"
        fill="none"
        focusable={false}
        height={20}
        meetOrSlice={0}
        minX={0}
        minY={0}
        stroke="#666"
        strokeWidth={2}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 20,
              "width": 20,
            },
          ]
        }
        vbHeight={24}
        vbWidth={24}
        width={20}
      >
        <RNSVGGroup
          fill={null}
          propList={
            [
              "fill",
              "stroke",
              "strokeWidth",
            ]
          }
          stroke={
            {
              "payload": 4284900966,
              "type": 0,
            }
          }
          strokeWidth="2"
        >
          <RNSVGPath
            d="M13.5 10.5V6.75a4.5 4.5 0 119 0v3.75M3.75 21.75h10.5a2.25 2.25 0 002.25-2.25v-6.75a2.25 2.25 0 00-2.25-2.25H3.75a2.25 2.25 0 00-2.25 2.25v6.75a2.25 2.25 0 002.25 2.25z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          {
            "color": "#444",
            "display": "none",
            "fontSize": 16,
            "marginLeft": 8,
          }
        }
      >
        Lock
      </Text>
    </View>
  </View>
</View>
`;
